tests/CloudRequestTests.php

```diff
@@ -36,17 +36,16 @@
 
 use PHPUnit\Framework\TestCase;
 
-class CloudRequestEngineTests extends TestCase
+class CloudRequestTests extends TestCase
 {
     public function testCloudRequestEngine()
     {
         $params = array("resourceKey" => $_ENV["RESOURCEKEY"]);
 
         if ($params["resourceKey"] === "!!YOUR_RESOURCE_KEY!!") {
             $this->fail("You need to create a resource key at " .
-            "https://configure.51degrees.com and paste it into the " .
-            "phpunit.xml config file, " .
-            "replacing !!YOUR_RESOURCE_KEY!!.");
+            "https://configure.51degrees.com and export it " .
+            "before running tests");
```

Comment:
Would be better to say explicitly that the environment variable `RESOURCEKEY` needs to be set to the resource key. The term export may not be as clear to everyone

Reply:
Done, PTAL.